tags/state/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@marko-tags/state",
+  "description": "State manipulation in your template",
+  "version": "0.1.0",
+  "author": "Andrew Gliga <agliga@ebay.com>",
+  "bugs": "https://github.com/marko-js/tags/issues/new?template=Bug_report.md",
+  "files": [
+    "transformer.js",
+    "marko.json"
+  ],
+  "homepage": "https://github.com/marko-js/tags/tree/master/tags/subscribe",
+  "keywords": [
+    "marko",
+    "state"
+  ],
+  "license": "MIT",
+  "peerDependencies": {
+    "marko": "^4.15.0"
+  },
```

Comment:
This needs to be bumped to whatever version of Marko gets released with your state fixes.